src/main/java/com/cairn/waypoint/dashboard/endpoints/homeworkresponse/dto/SimplifiedHomeworkResponseDto.java

```diff
@@ -0,0 +1,42 @@
+package com.cairn.waypoint.dashboard.endpoints.homeworkresponse.dto;
+
+import com.cairn.waypoint.dashboard.entity.HomeworkResponse;
+
+import java.time.LocalDateTime;
+
+public class SimplifiedHomeworkResponseDto {
+    private Long id;
+
+    public String getResponse() {
```

Comment:
The structure of this class is a bit off. You have properties interspersed throughout with methods implemented within. Could we refactor to use lombok instead and keep our properties at the top of the class.